phys/module_ra_rrtmg_lw.F

```diff
@@ -11894,18 +11893,25 @@ SUBROUTINE RRTMG_LWRAD(                                        &
 
 !ccc Read time-varying trace gases concentrations and interpolate them to run date.
 !
-#ifdef CLWRFGHG
-
-   CALL read_CAMgases(yr,julian,"RRTMG",co2,n2o,ch4,cfc11,cfc12)
-
+IF(xtime.eq.0 .or. (mod(xtime,1440.).eq.0)) THEN
+ IF(ghg_input.eq.1) then
```

Comment:
@smileMchen Maybe the above two IFs should be swapped? Chech ghg_input first, before compute mod(xtime,1440.).

Reply:
@weiwangncar 
Wei,
I don't feel like to swap the two IFs. This is because logically, we want to activate ghg_input =1  (1) only at the beginning of the model run, and (2) only on a daily basis.